main.c

```diff
@@ -0,0 +1,620 @@
+char option;
+int count=3;
+
+
+FILE *fp;
+fp=fopen("abc.txt","r");
+fscanf(fp,"%s",pwd);
+
+
+ENTER_PASSWARD:
+printf("\n\n\n\n\n\n\n\n\n\n\n\n\n \t\t\t\t\tPLEASE ENTER THE LOGIN PASSWORD: ");
+    int p=0;
+    do{
+        pd[p]=getch();
+        if(pd[p]!='\r')
+		    {
+            printf("*");
+            }
+        p++;
+      }while(pd[p-1]!='\r');
+       pd[p-1]='\0';
+
+
+// Compare Enter passward with inbuild passward
+if(strcmp(pwd,pd)==0)
+{
+    printf("\n\n \t\t\t\t\tWELCOME........LOGIN IN SUCESSFULLY ");
+    printf("\n\n \t\t\t\t\tPLEASE WAIT . . . ");
+
+    Sleep(3000);
+    system("cls");
+
+
+}
+else{
+	count=count-1;
+	if (count!=0)
+	{
+		printf("\n\n \t\t\t\t\tINCORRECT LOGIN Attemp left : %d",count);
+		printf("\n\n \t\t\t\t\tPLEASE WAIT . . . ");
+		Sleep(3000);
+		system("cls");
+	    goto ENTER_PASSWARD;
+
+	}
+    else
+	   printf("\n\n \t\t\t\t\tINCORRECT LOGIN");
+	   exit(0);
+
+    }
+fclose(fp);
+
+}
+
+
+void resetpassword()
+{
+char pw[SIZE],c,rpw[SIZE];
+short int count,error,i=0;
+int option;
+
+Enter:
+printf("\n****************************** ");
+printf("\nENETR NEW PASSWORD: ");
+int p=0;
+    do{
+        pw[p]=getch();
+        if(pw[p]!='\r')
+		   {
+            printf("*");
+           }
+        p++;
+      }while(pw[p-1]!='\r');
+     pw[p-1]='\0';
+
+length=strlen(pw);
+FILE *fp;
+ if( error=checkValid(pw) )
+ {
+   printf("\n\nIts a valid password");
+   checkStrength(pw);
+
+printf("\nRECONFIRM PASSWORD: ");
+   while( (c=getch()) !='\r')
+      {
+        rpw[i]=c;
+        putchar('*');
+        i++;
+      }
+ rpw[i]='\0';
+ length=strlen(rpw);
+
+//Compair both the Entered and RECONFIRM Passward
+   if(strcmp(pw,rpw)==0)
+   {
+       fp=fopen("abc.txt","w");
+       fprintf(fp, "%s", rpw);
+       fclose(fp);
+       printf("\n\n*******************************");
+       printf("\nPASSWORD RESET SUCESSFULLY");
+       printf("\n*******************************");
+       system("cls");
+
+   }
+   else{
+    printf("\nERROR: Password does not match");
+	count_reset=count_reset-1;
+	if (count_reset!=0)
+	{
+		printf("\n\n INCORRECT LOGIN Attemp left : %d",count_reset);
+
+	    resetpassword();
+	}
+    else
+	   printf("\n\n \t\t\t\t\tMore then 3 attemps Cannot Rest passward");
+
+    }
+
+ }
+ else
+    goto Enter;
+}
+
+short int checkValid(char *x)
+{
+    alphabetCount=0;
+    digitCount=0;
+
+if(length<6)
+{
+printf("\n Password should be atleast 6 characters wide");
+return 0;
+ }
+
+  while((*x)!='\0')
+    {
+     if( isalpha(*x))
+      alphabetCount++;
+     else if( isdigit(*x) )
+      digitCount++;
+     else if( isascii(*x) )
+      symbolCount++;
+
+     x++;
+    }
+
+ if(alphabetCount==length) 
+ {                         
+  printf("\n ERROR: Password should contain both alphabets and digits");
+  return 0;
+ }
+ else if(digitCount==length) 
+ {                           
+  printf("\n ERROR: Password should contain both alphabets and digits");
+  return 0;
+ }

```

Comment:
it is better to check if alphabet and digit are present by a boolean instead of counting length